### PR TITLE
[os maintenance] add a cleanup step

### PR DESCRIPTION
**Pull request by kayiwa**
include a tidy up step to the maintenance

### CODE REVIEW

File: playbooks/os_updates.yml [hunk 1]
```diff
@@ -32,16 +32,21 @@
...
     - name: Ubuntu | remove dependencies that are no longer required
       ansible.builtin.apt:
         autoremove: true
       when: ansible_os_family == "Debian"
 
+    - name: Ubuntu | remove dependencies that are no longer required
+      ansible.builtin.apt:
+        autoremove: true
+      when: ansible_os_family == "Debian"
+
```
Comment (acozine) [inline playbooks/os_updates.yml:49]: This looks like a duplicate of the task above. Are `autoremove` and `clean` the same thing?

**Review by acozine — COMMENTED** [review]
This might just be copy-pasta.

File: playbooks/os_updates.yml (hunk 1 above)
Comment (kayiwa) [reply playbooks/os_updates.yml:49]: `autoremove` automatically removes dangling software `clean` clears `var/cache/apt/archives/` and `/var/cache/apt/archives/partial/`

Comment (acozine) [reply playbooks/os_updates.yml:49]: Yeah, I meant in ansible-speak. Line 42 already says `autoremove`. I'm thinking line 47 should say `clean`?

Comment (kayiwa) [reply playbooks/os_updates.yml]: 🤦🏿‍♂️

**Review by acozine — APPROVED** [review]
Looks good, thanks @kayiwa !

**Comment by acozine** [comment]
When I look in [launchpad.net](https://launchpad.net/~ondrej/+ppa-packages) I don't immediately see a PPA for `php` - most of them have version numbers (`php7.4`, `php8.0`) or are bundled with other things (`php-redis`, `php-yac`). Do we need a different repo name?

**Comment by acozine** [comment]
Rebased this branch - I hope that will fix the CI error.

**Pull request merged**